### PR TITLE
wishlist screen added

### PR DESCRIPTION
**Pull request by rajkishancode**

### CODE REVIEW

File: screens/wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+<body>
+    <header>
+        <nav class="navigation-bar">
+
+            <div class="nav-section">
+
+                <div class="nav-logo nav-section-items">
+                    <a class="navigation-logo"><img src="/assets/calmkart_logo.webp" alt="calmkart logo" /></a>
+                </div>
+
```
Comment (Aditi0505) [inline screens/wishlist/wishlist.html]: Can fix spacing here.

File: screens/wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <main class="wishlist mb-10">
+        <h3 class="h3 text-center mt-10 mb-6">Wishlist</h3>
+        <div class="auth-container">
+
+
```
Comment (Aditi0505) [inline screens/wishlist/wishlist.html]: can remove extra spaces here.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. Rest looks good.

**Review by hiteshverma27 — APPROVED** [review]
use a code formatter before pushing rest looks good

File: screens/wishlist/wishlist.html (hunk 1 above)
Comment (rajkishancode) [reply screens/wishlist/wishlist.html]: okay

File: screens/wishlist/wishlist.html (hunk 2 above)
Comment (rajkishancode) [reply screens/wishlist/wishlist.html]: okay will do that.

**Comment by rajkishancode** [comment]
> use a code formatter before pushing rest looks good

okay

**Pull request merged**